src/articles/infrastructure/persistence/relational/relational-persistence.module.ts

```diff
@@ -14,14 +16,16 @@ import { ArticleRelationalRepository } from './repositories/article.repository';
       ArticleEntity,
       ArticleFavoriteEntity,
       UserFollowEntity,
+      ClapEntity,
     ]),
   ],
   providers: [
     {
       provide: ArticleAbstractRepository,
       useClass: ArticleRelationalRepository,
     },
+    ClapRelationalRepository,
   ],
-  exports: [ArticleAbstractRepository],
+  exports: [ArticleAbstractRepository, ClapRelationalRepository],
```

Comment:
Clap repository should be exported from claps module. Or am I missing something?

Reply:
Really appreciate that, Again My bad, but fixed that